libminifi/include/c2/C2Agent.h

```diff
@@ -57,7 +57,8 @@ class C2Agent : public state::UpdateController, public state::response::Response
 
   C2Agent(const std::shared_ptr<core::controller::ControllerServiceProvider> &controller, const std::shared_ptr<state::StateMonitor> &updateSink, const std::shared_ptr<Configure> &configure);
 
-  virtual ~C2Agent() {
+  ~C2Agent() noexcept override {
```

Comment:
In libminifi code we currently don't use override, as there were some problems with it on older systems and it has not yet been thoroughly verified that it works OK now. (In extensions, it is okay.)

Reply:
fixed in e108d27